context/packages.sh

```diff
@@ -12,7 +12,7 @@ if [ "$EXTRA_APT_PACKAGES" ]; then
 fi
 
 if [ "$EXTRA_YUM_PACKAGES" ]; then
-    if ! grep -i "centos" /etc/os-release > /dev/null; then
+    if ((! grep -i "centos" /etc/os-release > /dev/null ) or (! grep -i "rocky" /etc/os-release > /dev/null )); then
```

Comment:
```suggestion
    if ! grep -iE "centos|rocky" /etc/os-release > /dev/null; then
```

The code here isn't valid bash syntax.